packages/quantic/scripts/build/deploy-community.ts

```diff
@@ -0,0 +1,170 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import {
+  sfdx,
+  SfdxListOrgsResponse,
+  SfdxPublishCommunityResponse,
+} from './util/sfdx';
+
+interface Options {
+  alias: string;
+  configFile: string;
+  community: {
+    name: string;
+    path: string;
+    template: string;
+  };
+}
+
+let step = 0;
+const totalSteps = 6;
+
+const log = (currentStep: number, message: string) =>
+  console.log(`[${currentStep}/${totalSteps}] ${message}`);
+
+const scratchOrgExists = async (options: Options): Promise<boolean> => {
+  const response = await sfdx<SfdxListOrgsResponse>('force:org:list');
+
+  const org = response.result.scratchOrgs.find(
+    (o) => o.alias === options.alias
+  );
+
+  if (!!org && org.status !== 'Active') {
+    console.warn(
+      `Org ${options.alias} is found but status is not active. Status is ${org.status}.`
+    );
+  }
+
+  return !!org && org.status === 'Active';
```

Comment:
```suggestion
  const isOrgActive = !!org && org.status !== 'Active';

  if (isOrgActive) {
    console.warn(
      `Org ${options.alias} is found but status is not active. Status is ${org.status}.`
    );
  }

  return isOrgActive;
```

Reply:
I went with something slightly different than you suggested. Let me know what you think of it.

    const isOrgFound = !!org;
    const isOrgActive = isOrgFound && org.status === 'Active';

    if (isOrgFound && !isOrgActive) {
        console.warn(
            `Org ${options.alias} is found but status is not active. Status is ${org.status}.`
        );
    }

    return isOrgActive;

Reply:
that's perfect